fairworkflows/fairworkflow.py

```diff
@@ -162,39 +162,6 @@ def first_step(self, step: FairStep):
         self.set_attribute(namespaces.PWO.hasFirstStep, rdflib.URIRef(step.uri))
         self._add_step(step)
 
-    @property
```

Comment:
Fair enough, as we discussed in the meeting earlier. Although I think we will need this back eventually, with some sort of 'connector' triples, perhaps. Maybe:
```
(workflow, has:varbinding, bindinguri)
(bindinguri, rdf:type, somenamespace:VarBinding)
(bindinguri, from, fairvariableuriA)
(bindinguri, from, fairvariableuriB)
```
(if you'll forgive the horrific abuse of notation above). I'm thinking of how e.g. CWL does it, where the workflow specifies that input A of step 2 comes from output B of step 2. I can check with Tobias/Remzi on this matter though since they're the semantic experts.

Reply:
Yes, I agree we need it back eventually. Indeed something like you propose makes sense to me.